Web/ForgeTrust.Runnable.Web.RazorWire/README.md

```
@@ -4,13 +4,20 @@ RazorWire lets ASP.NET Core MVC apps update UI by returning Razor fragments from
-This quickstart assumes you are in a clone of this repository with the .NET 10 SDK installed. The public package install matrix is still being finalized for v0.1, so the fastest way to feel RazorWire today is the in-repo sample.
+Runnable has not published the public `v0.1` package set yet, so the copy-paste path today is repo-local:
+
+1. Clone this repository and use the .NET 10 SDK.
+2. Run the MVC sample:
 
 ```bash
 dotnet run --project examples/razorwire-mvc/RazorWireWebExample.csproj
 ```
 
-Open the URL printed in the console and navigate to `/Reactivity`, wait for the `Permanent Island` card to load, then click the `+` button. The `Instance Score` and `Session Score` update in place without a full-page reload.
+3. Open the URL printed in the console and navigate to `/Reactivity`.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)